kotlin/scene-layer/README.metadata.json

```diff
@@ -6,16 +6,25 @@
         "scene-layer.png"
     ],
     "keywords": [
```

Comment:
I'm not too sure if indentation has any affect on reading of the json file, but for neatness, we should ensure indents are consistent throughout please